README.adoc

```diff
@@ -158,25 +181,25 @@ kubernetes.yaml
 include::finish/kubernetes.yaml[]
 ----
 
-The readiness and liveness probes are configured for the containers running the `system` 
+The startup, liveness and readiness probes are configured for the containers running the `system` 
```

Comment:
The startup, liveness and readiness probes are configured for the containers running the `system` 
and `inventory` microservices.
--->
The startup, liveness and readiness probes are configured for the containers that are running the `system` 
and `inventory` microservices.